internal/image/image_check.go

```diff
@@ -0,0 +1,66 @@
+package image
+
+import (
+	"github.com/OpenNMS/opennms-operator/api/v1alpha1"
+	"github.com/go-co-op/gocron"
+	"github.com/go-logr/logr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+const (
+	None             = "none"
+	Now              = "now"
+	Autoupdate       = "autoupdate"
```

Comment:
Suspect this is no longer needed

Reply:
I figured. I'd removed it from other places but forgot to here.